ui/src/utils/constants.ts

```diff
@@ -114,3 +114,9 @@ export const Languages = [
     value: 'de',
   },
 ]
+
+export const LanguagesMap = {
+  zh: '简体中文',
+  en: 'English',
+  de: 'Deutsch',
```

Comment:
We have optimized the code, you can add a new language declaration here. @virtually-stray